src/platform/monitoring/DowntimeNotification/tests/helpers.unit.spec.js

```diff
@@ -23,6 +23,18 @@ const pastDowntime = {
   },
 };
 
+const futureDowntimeForSameService = {
+  attributes: {
+    externalService: 'evss',
+    startTime: moment()
```

Comment:
just fwiw I think we're trying to move away from `moment` in general towards DateFNS. No expectation of converting this test though!

Reply:
Yeah I wanted to maintain consistency with the other tests. I'm assuming that at some point someone will need to go in and update all of these to not use `moment`